app/models/service_template.rb

```diff
@@ -450,6 +451,10 @@ def self.display_name(number = 1)
     n_('Service Catalog Item', 'Service Catalog Items', number)
   end
 
+  def my_zone
+    zone&.name
```

Comment:
Are we ever expecting this to be `nil`?

Reply:
Right, the zone of a service template is optional.